player.rb

```diff
@@ -0,0 +1,30 @@
+# Class for a blackjack player to keep track of cards in hand and total value
+class Player
+  attr_reader :hand, :total
+  attr_accessor :hand, :total
+
+  def initialize
+    @hand = []
+    @total = calculate_total
```

Comment:
I think @total might be superfluous here. Storing a computed value as local state can be useful when the calculation is expensive or should be allowed to get out of sync with its determinants, but in this case it's fast and we always want to know what the current total is, and we have the list of cards in the hand available to us to track the state. I'd probably just rename the `calculate_total` method to `total` and ask for that whenever I wanted to know the value of the hand. Ruby represents a bit of a different way of thinking about the difference between values and methods, in that there really isn't one.